src/cli/tests/test_evm_bytes_to_python.py

```diff
@@ -10,11 +10,11 @@
 
 basic_vector = [
     "0x60008080808061AAAA612d5ff1600055",
-    'Op.PUSH1("0x00") + Op.DUP1 + Op.DUP1 + Op.DUP1 + Op.DUP1 + Op.PUSH2("0xaaaa") + Op.PUSH2("0x2d5f") + Op.CALL + Op.PUSH1("0x00") + Op.SSTORE',  # noqa: E501
+    "Op.PUSH1[0x0] + Op.DUP1 + Op.DUP1 + Op.DUP1 + Op.DUP1 + Op.PUSH2[0xaaaa] + Op.PUSH2[0x2d5f] + Op.CALL + Op.PUSH1[0x0] + Op.SSTORE",  # noqa: E501
```

Comment:
The result of push2(1) is still 610001 right?

Also what happens if push1(257)?

And one more thing that 0x0123 in python is really confusing as it is integer, but in evm we think of 0x0123 as bytes or hash. And thus makes me want to write in python push2(0x0001)

Reply:
> The result of push2(1) is still 610001 right?

Yes that is correct, I don't think we could remove that feature

> Also what happens if push1(257)?

It will fail, if the number overflows the data-portion length of any opcode, it will throw an error.

> And one more thing that 0x0123 in python is really confusing as it is integer, but in evm we think of 0x0123 as bytes or hash. And thus makes me want to write in python push2(0x0001)

If you write `0x0123` it will work, but any linter will automatically change it to remove the leading zeros.